README.md

```diff
@@ -1,14 +1,29 @@
 # artsy-wwwify
 A tiny app that 301 redirects requests to www.artsy.net
 
-### Deploy
-
-This app is deployed on Amazon OpsWorks. [Click here to deploy](https://console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/7db728d1-44a8-49e0-8362-92e4723c898e/deployments/deploy-app).
+### Meta
+* State: production
+* Production: https://artsy.net
+* Staging: N/A
+* GitHub: https://github.com/artsy/artsy-wwwify
+* CI: [CircleCI](https://circleci.com/gh/artsy/artsy-wwwify); merged PRs to `master` are automatically built and deployed to staging. [Make a pull request from `staging` to `release`](https://github.com/artsy/artsy-wwwify/compare/release...staging?expand=1) to trigger a release to production.
+Point People: @joeyAghion
 
 ### Development
 
-    npm run dev
+    npm install
```

Comment:
Since 2016 we've been using `yarn` instead of `npm`.

Reply:
I recognize that but was trying to limit this PR to necessary changes. I'm happy to give it a try though... is `yarn` already available in the base image, or would it require other set-up?